images/vep/Dockerfile

```diff
@@ -45,17 +45,19 @@ RUN rm $VEP_SHARE/TissueExpression.pm \
   $VEP_SHARE/svm.pl \
   $VEP_SHARE/utr_splice.pl && \
   git clone https://github.com/populationgenomics/loftee_38.git && \
-    cp -r loftee_38/* $MAMBA_ROOT_PREFIX/share/ensembl-vep-$VERSION-1 \
-    && rm -rf loftee_38
+    cp -r loftee_38/* $VEP_SHARE \
+    && rm -rf loftee_38 && \
 
-# Install gcsfuse to mount VEP cache files
-RUN GCSFUSE_REPO=gcsfuse-$(lsb_release -c -s) && \
+# Install gcsfuse to mount VEP cache files"
+
```

Comment:
There's now a few newlines like these, maybe get rid of any newlines in a single `RUN`?